docs/modules/user-guide/partials/proc_adding-a-command-section-to-a-devfile.adoc

```diff
@@ -0,0 +1,75 @@
+[id="proc_adding-a-command-section-to-a-devfile_{context}"]
+= Adding a command section to a devfile
+[role="_abstract"]
+
+Specify the commands you want to execute in your workspace.
```

Comment:
s/execute/run throughout.
IBM SG:
```
execute verb
Use only if a simpler term, such as "run", "issue", "start", or "enter" is inappropriate.
```
For many people, the word execute has strong associations with killing.